base/file.jl

```diff
@@ -730,8 +738,12 @@ end
 Apply the function `f` to the result of [`mktempdir(parent; prefix)`](@ref) and remove the
 temporary directory all of its contents upon completion.
 
+<<<<<<< HEAD
 !!! compat "Julia 1.2"
     The `prefix` keyword argument was added in Julia 1.2.
+=======
+See also: [`mktemp`](@ref), [`mkdir`](@ref).
+>>>>>>> another batch of see also additions
```

Comment:
An error I just spotted:
```suggestion
!!! compat "Julia 1.2"
    The `prefix` keyword argument was added in Julia 1.2.

See also: [`mktemp`](@ref), [`mkdir`](@ref).
```